.env.example

```diff
@@ -56,3 +56,5 @@ SMTP_LOGIN=
 SMTP_PASSWORD=
 EMAIL_FROM_NAME=
 EMAIL_FROM=no-reply@example.com
+
+REGISTRATION_MESSAGE_TO_SIGN=I register as D-REC user
```

Comment:
why it's env? Strange

Reply:
This is the textual string that the user signs with his private key to prove public key ownership.